module/documents/actor.mjs

```diff
@@ -118,6 +118,86 @@ export default class ActorEd extends Actor {
     this.#processRoll( roll );
   }
 
+  /**
+   * @description                 Roll an Ability. use {@link RollPrompt} for further input data.
+   * @param {object} ability      ability.
+   * @param {object} options      Any additional options for the {@link EdRoll}.
+   */
+  async rollAbility( ability, options = {} ) {
+    const attributeStep = this.system.attributes[ability.system.attribute].step;
+    const abilityFinalStep = attributeStep + ability.system.level;
+    const difficulty = await this.getTarget( ability );
+    const edRollOptions = new EdRollOptions( {
+      rollType: "action",
```

Comment:
Depending on how we implement the combat workflow, this could also be damage (for things like Claw Shape, Body Control, etc.).
See Review of #636.